query/src/main/java/io/keen/client/java/KeenQueryConstants.java

```diff
@@ -3,10 +3,10 @@
 /**
  * Commonly used constants in the HTTP query request
  *
- * @author claireyoung
+ * @author claireyoung, baumatron, masojus
  * @since 1.0.0, 05/18/15
  */
-public class KeenQueryConstants {
+class KeenQueryConstants {
```

Comment:
Comment: Changing this from `public` to default is, technically speaking, a breaking change (if somebody was, hypothetically, referencing constants from this class, their code would break). We can consider doing it anyway, but we should be careful about this kind of thing.